src/index.js

```diff
@@ -117,38 +119,53 @@ $('#submit_team-member').on('click', function(event) {
   }
 })
 
-$('#submit_book-room-button').on('click', function(event) {
+// PORTAL PAGES EVENT LISTENERS
+
+$('#button_find-room-js').on('click', function(event) {
   $('.div_available-rooms').remove();
+  $('.error_no-rooms').addClass('hide');
   selectedDate = formatSelectedDate($('#select-date').val());
   let roomType = $('#select-room-type').val();
   if (selectedDate !== ''
-    && event.target.parentNode.parentNode.classList.contains('main_test-class')) {
-      displayAvailableRooms(selectedDate, roomType, '.main_guest-page');
+    && event.target.parentNode.classList.contains('main_guest')) {
+      displayAvailableRooms(selectedDate, roomType, '.main_guest');
   } else if (selectedDate !== ''
-    && event.target.parentNode.parentNode.classList.contains('main_manager-page')) {
+    && event.target.parentNode.parentNode.classList.contains('main_manager')) {
       displayAvailableRooms(selectedDate, roomType, '.div_right-main')
   } else {
-    $('#submit_book-room-button, .button_new-search, .error_no-rooms')
-      .toggleClass('hide');
+    $('.error_no-rooms').removeClass('hide');
   }
 })
 
-$('.button_new-search').on('click', function() {
+$('.button_clear-search').on('click', function() {
+  $('.div_available-rooms').remove();
   $('#select-date').val('');
   $('#select-room-type').val('any');
-  $('#submit_book-room-button, .button_new-search, .error_no-rooms')
-    .toggleClass('hide');
+  $('.error_no-rooms').addClass('hide');
 })
 
-$('.main_guest-page').on('click', '.div_available-rooms', function(event) {
+// CREATE AND POST NEW BOOKING FOR USER
+$('.main_portal-page').on('click', '.div_available-rooms', function(event) {
   event.preventDefault();
-  let postBody = user.makeBooking(selectedDate,
-    event.target.closest('.div_available-rooms').dataset.roomnumber);
-  postNewBooking(postBody);
-  $(event.target.closest('.div_available-rooms')).remove();
+  let postBody;
+  if (event.target.closest('.div_available-rooms').parentNode.classList.contains('main_guest')) {
```

Comment:
all of these seem like odd mixes of js and jquery, maybe see if this works:

$(event.target).hasClass('main_guest') could help with a lot of these

Reply:
Thanks Matt.  You are absolutely right on this. I really struggled to get these to cooperate.   I think I'll take a brain break from it and try to clean it up tomorrow with fresh eyes.